runtime/browser/ui/native_app_window_views.cc

```diff
@@ -158,6 +180,15 @@ void NativeAppWindowViews::Close() {
   window_->Close();
 }
 
+void NativeAppWindowViews::ExitApplication() {
+  auto app_service =
```

Comment:
How about this?

void Application::Terminate()
void Runtime::ExitApplication()
void RuntimeUIDelegateDesktop::OnApplicationExitRequested() 
void NativeAppWindowViews::ExitApplication()